articles/user-guide/libraries/standard/diagnostics.md

```diff
@@ -159,7 +160,7 @@ By contrast, the referenced assertion calls each operation exactly once, such th
 Both of these tests are useful in ensuring the correctness of quantum programs.
 
 
-## Further Reading ##
+## Next steps
 
-- <xref:microsoft.quantum.guide.testingdebugging>
-- <xref:microsoft.quantum.diagnostics>
+- Read more about unit tests, facts and assertions, dump functions, and debugging in Visual Studio in <xref:microsoft.quantum.guide.testingdebugging>.
```

Comment:
That's a good point; can we broaden `testingdebugging` to not so strongly privilege Visual Studio, given that's no longer the strict majority of our users?